kolibri/core/assets/src/core_app_constructor.js

```diff
@@ -44,15 +48,22 @@ function Lib() {
  */
 module.exports = function CoreApp() {
   this.lib = new Lib();
+  this.resources = new ResourceManager(this);
   const mediator = new Mediator();
 
+  this.resources.registerResource('contentmetadata', 'id', ContentMetaDataResource);
```

Comment:
Since `idName` seemed to be an optional field (and in practice, 95% of our resources will use that default), can it be omitted here?